dashboard/modules/job/job_manager.py

```diff
@@ -517,15 +521,19 @@ async def submit_job(
         # returns immediately and we can catch errors with the actor starting
         # up.
         try:
+            scheduling_strategy = "DEFAULT"
```

Comment:
I don't think you need to specify default here
https://sourcegraph.com/github.com/ray-project/ray@c9572664b02345a8c8ed517950b28e792f4f4940/-/blob/python/ray/remote_function.py?L348

Maybe just None?